case-lib/hijack.sh

```diff
@@ -30,12 +30,23 @@ function func_exit_handler()
             func_lib_start_log_collect 1
             sleep 1s
         }
-        local loggerBin; loggerBin=$(basename "$SOFLOGGER")
-        sudo pkill -9 "$loggerBin" 2>/dev/null || {
+
+        local loggerBin wcLog; loggerBin=$(basename "$SOFLOGGER")
+        # logfile is set in a different file: lib.sh
+        # shellcheck disable=SC2154
+        wcLog=$(wc -l "$logfile")
+        dlogi "nlines=$wcLog"
```

Comment:
maybe a more readable log here? This looks like a cmd.

Reply:
In the next PR I will implement a sanity check on the number of lines. For now this is not great looking but it's readable and I'd rather not write and test code that will change in the next PR.